tests/run.sh

```diff
@@ -1,30 +1,31 @@
 #!/bin/bash
 set -x -e
 RUN_S3=""
+MENDER_BRANCH=$(cd .. && python extra/release_tool.py --version-of mender)
+
+if [[ $? -ne 0 ]]; then
+    echo "Failed to determine mender version using release_tool.py"
+    exit 1
+fi
+
+echo "Detected Mender branch: $MENDER_BRANCH"
 
 function get_requirements() {
     # Download what we need.
     mkdir -p downloaded-tools
-    curl "https://d25phv8h0wbwru.cloudfront.net/${TEST_BRANCH}/tip/mender-artifact" \
+    curl "https://d25phv8h0wbwru.cloudfront.net/${MENDER_BRANCH}/tip/mender-artifact" \
```

Comment:
Careful, this is a different version, you need to get `--version-of mender-artifact` for this.